tools/tpm2_nvwrite.c

```diff
@@ -39,10 +39,10 @@
 
 #include <sapi/tpm20.h>
 
+#include "../lib/tpm2_session.h"
```

Comment:
Do you really need a relative path here? I thought this wasn't needed since lib is included in the list of directories to search for header files.

Reply:
Nope eclipse code formatter does some weird things